packages/schematics/src/tslint/nxEnforceModuleBoundariesRule.spec.ts

```diff
@@ -340,10 +340,10 @@ describe('Enforce Module Boundaries', () => {
             tags: [],
             implicitDependencies: [],
             architect: {},
-            files: [`libs/mylib/src/main.ts`, `libs/mylib/other/index.ts`],
+            files: [`libs/mylib/src/main.ts`, `libs/mylib/src/other/index.ts`],
```

Comment:
Why is this change necessary? Isn't the code below makes it work without src too?

Reply:
To be honest, I don't know why I changed that test 🤨 I reverted that change and instead added an explicit test case for my change.

But that highlights another thing, that I was already wondering about, when I made the change. Under what circumstances do we have a situation as described in the `should error when relatively importing another library` test case? e.g. an import to `/other` that resolves to `/other/src/index.ts`. When I create a new Nx Workspace, that wouldn't even work with the tsconfig as it is generated. And that begs the question, why this is even part of the existing implementation of the rule:

https://github.com/nrwl/nx/blob/c5fdf1c6562c628387f1efa57de6d56638d7d336/packages/schematics/src/tslint/nxEnforceModuleBoundariesRule.ts#L289

Reply:
You are right. The TS compiler will fail, so this doesn't seem to be necessary. 

I think that we should have just checked that you aren't importing outside your library. It shouldn't even matter if your target file is in a library or not. It's a bit more restricting but simpler to understand.